test/lib/code_corps/stripe_service/stripe_connect_account_service_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule CodeCorps.StripeService.StripeConnectAccountServiceTest do
+  use ExUnit.Case, async: true
+
+  use CodeCorps.ModelCase
+
+  alias CodeCorps.StripeConnectAccount
+  alias CodeCorps.StripeService.StripeConnectAccountService
+
+  describe "create" do
+    test "creates a StripeConnectAccount" do
+      organization = insert(:organization)
+
+      attributes = %{"country" => "US", "organization_id" => organization.id}
+
+      {:ok, %CodeCorps.StripeConnectAccount{} = connect_account} =
+              StripeConnectAccountService.create(attributes)
+
+      assert connect_account.country == "US"
+      assert connect_account.organization_id == organization.id
```

Comment:
I tried putting `"managed => true` in the attributes and then doing `assert connect_account.managed(true) == :managed` but it looks like managed is still false. Is there some other way to set it to true?